forge-gui/res/cardsfolder/i/immobilizer_eldrazi.txt

```diff
@@ -3,6 +3,7 @@ ManaCost:1 R
 Types:Creature Eldrazi Drone
 PT:2/1
 K:Devoid
-A:AB$ PumpAll | Cost$ 2 C | ValidCards$ Creature.powerLTtoughness | KW$ HIDDEN CARDNAME can't block. | SpellDescription$ Each creature with toughness greater than its power can't block this turn.
```

Comment:
`HIDDEN CARDNAME can't block.` needs to be removed from other cards too

Reply:
I know but too many for now especially since AI logic depends on it

Reply:
Yeah, CARDNAME can't block is a tricky case (as outlined in #3790) - the AI can only detect inability to block through this pseudo-keyword. The (proper?) static ability application of can't block currently causes issues for the AI that I haven't yet found a way to resolve. Changing all the cards to a CantBlock static will require a certain rework of the AI since otherwise it'll break AI determining whether something can or can't block completely.

Reply:
(Addendum: assistance is definitely needed with that, as I've tried a few different things in the past but none of them did the trick :/ )